File: app/src/main/java/com/meiji/daily/injector/component/PostsListComponent.java
package com.meiji.daily.injector.component;

import com.meiji.daily.injector.module.PostsListModule;
import com.meiji.daily.mvp.postslist.PostsListView;

import dagger.Component;

/**
 * Created by Meiji on 2017/7/10.
 */
@Component(modules = PostsListModule.class)
public interface PostsListComponent {

    void inject(PostsListView view);

}

File: app/src/main/java/com/meiji/daily/module/postslist/PostsListComponent.kt
package com.meiji.daily.module.postslist

import com.meiji.daily.di.component.AppComponent
import com.meiji.daily.di.scope.FragmentScoped

import dagger.Component

/**
 * Created by Meiji on 2017/12/28.
 */
@FragmentScoped
@Component(modules = [(PostsListModule::class)], dependencies = [(AppComponent::class)])
interface PostsListComponent {

    fun inject(postsListView: PostsListView)
}
